README.md

```diff
@@ -24,21 +24,21 @@ $ pip install -U proofdock-chaos-kit
 
 ## Usage
 
-To use the features ...
+The Proofdock Chaos CLI extends the [Chaos Toolkit][chaostoolkit] and is used on the command line. The several Proofdock Chaos CLI commands are explained in detail at the [Proofdock documentation homepage][proofdock_docs_chaoscli]. 
 
 ## Configuration
 
-The [Proofdock Chaos Kit][proofdock_docs_chaoskit] uses the [Chaos Toolkit][chaostoolkit] under the hood. The Proofdock Chaos Kit expects that you have an API url and a proper API token that allows you to authenticate against the Proofdock chaos API.
+The Proofdock Chaos CLI uses the Chaos Toolkit under the hood. The Proofdock Chaos CLI expects that you have an API url and a proper API token that allows you to authenticate against the Proofdock cloud.
```

Comment:
Can we remove this statement "under the hood"? And keep to the statement that we extend chaostoolkit CLI?